dev/make/compiler_definitions/icx.mkl.32e.mk

```diff
@@ -18,29 +18,40 @@
 #  Intel compiler definitions for makefile
 #--
 
-PLATs.icx = lnx32e mac32e
```

Comment:
should mac32e be deleted? or just windows added?

Reply:
YEs, we dont have support icx on macos